src/demo-app/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Material2</title>
-  <base href="/">
+  <base href=".">
```

Comment:
Angular requires an explicit base tag, and [the docs recommend this value](https://angular.io/docs/ts/latest/guide/router.html).